### PR TITLE
Add prettier format script & run

### PR DESCRIPTION
**Pull request by tiaanduplessis**

### CODE REVIEW

File: src/h.js [hunk 1]
```diff
@@ -28,4 +28,3 @@ export default function(tag, data) {
       }
     : tag(data, children)
 }
-
```
Comment (zigomir) [inline src/h.js]: This tiny change somehow doesn't conform to https://github.com/hyperapp/hyperapp/blob/master/.editorconfig#L7 imo 🐰

**Comment by selfup** [comment]
going to pull down and take a look!

File: src/h.js (hunk 1 above)
Comment (selfup) [inline src/h.js]: It added a newline for me when I ran `yarn run format` after deleting the line on purpose.

But yes agreed, somehow this got missed :P

Comment (dodekeract) [inline src/h.js]: There still is a newline. It removed a duplicate newline.

Comment (selfup) [inline src/h.js]: Ahh yes the way github diffs. Makes sense 👍

**Comment by jbucaran** [comment]
Sorry for the delay! Thanks @tiaanduplessis! 👏

**Pull request merged**